src/test/shell/bazel/remote/remote_execution_http_test.sh

```diff
@@ -59,6 +59,24 @@ function tear_down() {
   rm -rf "${work_path}"
 }
 
+function test_remote_http_cache_flag() {
+  # Test that the deprecated --remote_http_cache flag still works.
+  mkdir -p a
+  cat > a/BUILD <<EOF
+genrule(
+  name = 'test-remote-http-cache',
```

Comment:
nit: call it `foo` or `gen` or so and name the output file after the target.

Reply:
done